<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Colors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Document</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graphics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mouse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plugins</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="colorsPage">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_bg_color">
           <property name="text">
            <string>Graph background color:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KColorButton" name="backgroundColor">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_entity_bg_color">
           <property name="text">
            <string>Entity background color:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KColorButton" name="entityBackgroundColor">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_selected_entity_bg_color">
           <property name="text">
            <string>Selected entity background color:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KColorButton" name="selectedEntityBackgroundColor">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_edge_color">
           <property name="text">
            <string>Edge color:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KColorButton" name="edgeColor">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_highlight_edge_color">
           <property name="text">
            <string>Highlight edge color:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KColorButton" name="highlightEdgeColor">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="chkSelectedEntityHasGradient">
           <property name="text">
            <string>Use gradient color on selected entities</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="chkColorBlindness">
           <property name="text">
            <string>Adjust for colorblindness</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="chkColorPattern">
           <property name="text">
            <string>Use patterns instead of solid colors</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="documentPage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Time in milisseconds for automatically saving a backup after a change on the document</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="autoSaveBackupIntervalMsecs">
           <property name="suffix">
            <string>  ms</string>
           </property>
           <property name="minimum">
            <number>1000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Automatic Backup Timer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Backups are saved in:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLabel" name="saveFolder">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="graphicsPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="VisualizationOptions">
            <attribute name="title">
             <string>Default Selection</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Default Tools</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QCheckBox" name="minimap">
                  <property name="text">
                   <string>Minimap</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="toolBox">
                  <property name="text">
                   <string>Information Box</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Default Values for Physical Load</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QCheckBox" name="showProviders">
                  <property name="text">
                   <string>Providers</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="showClients">
                  <property name="text">
                   <string>Clients</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Default Values for Logical Load</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QCheckBox" name="isARelation">
                  <property name="text">
                   <string>Load Inheritance Relationships (Is A)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="usesInTheInterface">
                  <property name="text">
                   <string>Load Uses in the Interface Relationships</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="usesInTheImplementation">
                  <property name="text">
                   <string>Load Uses In the Implementation Relationships</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="DefaultValues">
            <attribute name="title">
             <string>Default Values</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Entity Name Location</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="entityNamePos">
               <item>
                <property name="text">
                 <string>Top Left</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Top Right</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Bottom Left</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Bottom Right</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Class Limit</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="classLimit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Relation Limit</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="relationLimit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Zoom Level</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="zoomLevel">
               <property name="suffix">
                <string> %</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="showLevelNumbers">
               <property name="text">
                <string>Show Level Numbers</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Fonts">
            <attribute name="title">
             <string>Fonts</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Package Groups</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="KFontRequester" name="pkgGroupFont"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Packages</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="KFontRequester" name="pkgFont"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Components</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="KFontRequester" name="componentFont"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Classes</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="KFontRequester" name="classFont"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Structs</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="KFontRequester" name="structFont"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Enums</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="KFontRequester" name="enumFont"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mousePage">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Pan View With</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboPanModifier"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Zoom with mouse wheel and</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboZoomModifier"/>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="debugPage">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="debugContextMenu">
           <property name="text">
            <string>Enable Debug Context Menu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="enableDebugOutput">
           <property name="text">
            <string>Enable Debug Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="storeDebugOutput">
           <property name="text">
            <string>Store Debug Output</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="designPage">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="lakosianRules">
           <property name="text">
            <string>Force &quot;Large Scale C++ Development&quot; naming specifications</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>This setting makes the software strict on what kind of architecture it generates, following the book &quot;Large Scale C++ Development&quot; ideas. If you don't have the book, you can read a resume of the rules on the BDE Coding Standard. The tool will help you when mistakes are made.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showRedundantEdgesDefaultCheckbox">
           <property name="text">
            <string>Show redundant edges by default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hidePkgPrefixOnComponents">
           <property name="text">
            <string>Hide package prefix on components that follows &quot;Large Scale C++ Development&quot; naming specifications</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="invertHorizontalLvlLayout">
           <property name="text">
            <string>Invert direction on horizontal levelization layout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="invertVerticalLvlLayout">
           <property name="text">
            <string>Invert direction on vertical levelization layout</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>408</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pluginsPage">
        <layout class="QVBoxLayout" name="pluginsPageLayout">
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
